Description: snmp: do not die if there is no IPv6 support available
Bug: http://www.cups.org/str.php?L4109
Applied-Upstream: commit:10535

--- a/backend/snmp.c
+++ b/backend/snmp.c
@@ -253,7 +253,7 @@ main(int  argc,				/* I - Number of comm
 
 #ifdef AF_INET6
   if ((ipv6 = _cupsSNMPOpen(AF_INET6)) < 0)
-    return (1);
+    perror("DEBUG: Unable to create IPv6 socket");
 #else
   ipv6 = -1;
 #endif /* AF_INET6 */
